src/components/TodoBody.js

```diff
@@ -0,0 +1,30 @@
+import InputTodo from './ImputTodo';
+import TodoList from './TodoList';
+/* eslint-disable-next-line */
+function TodoBody() {
+  const todos = [
+    {
+      id: 1,
+      title: 'Setup development environment',
+      completed: true,
+    },
+    {
+      id: 2,
+      title: 'Develop website and add content',
+      completed: false,
+    },
+    {
+      id: 3,
+      title: 'Deploy to live server',
+      completed: false,
+    },
+  ];

```

Comment:
Make sure you persist tha data on localstorage. That will ensure that users will be able to perform all the CRUD operations. Note that depending on the project's requirement, you may also chose to store data in a database or a json file that can be run separately.